### PR TITLE
Added the cryto currency folder in the scripts folder

### LINKED ISSUE #17: Crypto Currency
### Describe the new script you want
In this script, we will see all the details of cryptocurrency at the instinct of time. Also, if we want to know about a particular crypto coin then it will add those details too. All these details will be stored in a CSV file.

I have used selenium and beautiful soup to do so.

## Do you want to work on this?
- [X] yes
- [ ] no

### PR DESCRIPTION
**Pull request by jayaganeshkumar**
# Description

Please include a general summary of the change and/or which issue it fixed. 

Fixes #17 

## Type of change

Please delete options that are not relevant.
- [X] New Script (non-breaking change which adds a new script to the repository)

## Has This Been Tested?

- [X] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [X] I have gone through the contributing guidelines before making this Pull Request.
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by soham4abc** [comment]
@jayaganeshkumar 
Run black locally on your machine to reformat your Python code!
The below article will help you doing so:
https://black.readthedocs.io/en/stable/getting_started.html#basic-usage

Cheers!

**Review by soham4abc — CHANGES_REQUESTED** [review]
Run `Black` Code formatter

**Review by OBITORASU — CHANGES_REQUESTED** [review]
Currently the chromedriver location is hardcoded. I think we should inform the user that they need a webdriver for their respective browser to run this script and also let them have control over the location instead of hardcoding it.

File: scripts/CryptoCurrency/main.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+options = webdriver.ChromeOptions()
+options.add_argument("--ignore-certificate-errors")
+options.add_argument("--ignore-ssl-errors")
+options.add_argument("--headless")
+options.add_argument("--hide-scrollbars")
+options.add_argument("--disable-gpu")
+options.add_argument("--log-level=3")
+
+# initialize the webdriver
+driver = webdriver.Chrome(executable_path=r"D:\chromedriver.exe", options=options)
```
Comment (OBITORASU) [inline scripts/CryptoCurrency/main.py:19]: Talking about this part in particular. Could you do something about it?

**Comment by OBITORASU** [comment]
I think you got me wrong. The approach should be, first we print out a verbose message telling the user that they must download the webdriver for this script to work. The if the webdriver is there then the user provides a path and if the user fails to provide a valid path the code exits with return code 1. Use this approach as reference.

**Comment by jayaganeshkumar** [comment]
> I think you got me wrong. The approach should be, first we print out a verbose message telling the user that they must download the webdriver for this script to work. The if the webdriver is there then the user provides a path and if the user fails to provide a valid path the code exits with return code 1. Use this approach as reference.

Can I write a GUI code to open file explorer which copies the path of the driver to the code?

**Comment by OBITORASU** [comment]
Yes if that is what you are comfortable with feel free. I just want you to let the user know that they need to download a webdriver for their browser, and allow them to set the path to it. :)

**Comment by jayaganeshkumar** [comment]
> Yes if that is what you are comfortable with feel free. I just want you to let the user know that they need to download a webdriver for their browser, and allow them to set the path to it. :)

check it now!!

**Comment by OBITORASU** [comment]
LGTM!

**Comment by jayaganeshkumar** [comment]
> LGTM! Ty for contributing

Label this as accepted.

**Comment by OBITORASU** [comment]
If it is merged they automatically consider it towards your progress, accepted label is for on-going PRs which are considered good. I will have it added regardless since you insist.

**Comment by jayaganeshkumar** [comment]
> If it is merged they automatically consider it towards your progress, accepted label is for on-going PRs which are considered good. I will have it added regardless since you insist.

Thanks, man. If you have any other ideas, I am ready to work with you.

**Pull request merged**